### ISSUE TITLE
servermap.py example not works

### ISSUE THREAD

**Issue by jaraco** [jaraco/irc#59]
```
#!bash

(AvsPmod)[nekmo@homura /tmp/irc/scripts]$ python servermap.py irc.rizon.net:6667 Nekmocom
Traceback (most recent call last):
  File "servermap.py", line 170, in <module>
    main()
  File "servermap.py", line 152, in main
    reactor = irc.reactor.Reactor()
AttributeError: 'module' object has no attribute 'reactor'
(AvsPmod)[nekmo@homura /tmp/irc/scripts]$ python2 servermap.py irc.rizon.net:6667 Nekmocom
Traceback (most recent call last):
  File "servermap.py", line 170, in <module>
    main()
  File "servermap.py", line 152, in main
    reactor = irc.reactor.Reactor()
AttributeError: 'module' object has no attribute 'reactor'
(AvsPmod)[nekmo@homura /tmp/irc/scripts]$ 
```

OS: Arch Linux.
Package version: irc==12.1.1

The error occurs inside (clean installation) and outside (system) of a virtualenv.

---
- Bitbucket: https://bitbucket.org/jaraco/irc/issue/59
- Originally reported by: [Nekmo](http://bitbucket.org/Nekmo)
- Originally created at: 2015-04-24T07:57:04.182

**Comment by jaraco** [comment]
Correct references to irc.client module, unintentionally replaced. Fixes #59.

→ <<cset 785d43181ef1>>

---

Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Released as 12.1.2. Thanks for the report.

---

Original comment by: Jason R. Coombs
